client/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from 'react'
+
+import './Card.scss'
+import { CardType } from '../../types/CardType'
+
+const cardToEmoji = (card: CardType) => {
+  let emoji
+  let label
+  switch (card) {
+    case CardType.EGG:
+      emoji = '🥚'
+      label = 'Egg'
+      break
+    case CardType.CHICKEN:
+      emoji = '🐔'
+      label = 'Chicken'
+      break
+    case CardType.FOX:
+      emoji = '🦊'
+      label = 'Fox'
+      break
+  }
+  return (
+    <span className='Card__Emoji' role='img' aria-label={label}>
+      {emoji}
+    </span>
+  )
+}
+
+export interface CardProps {
+  type: CardType
+  className?: string
+}
+
+const Card: FC<CardProps> = ({ type, className }) => {
+  const classes = 'Card__Container ' + className
+  return (
+    <div className={classes}>
+      <div className='Card__AspectRatio'>
+        <div className='Card__Content'>{cardToEmoji(type)}</div>
+      </div>
+    </div>
```

Comment:
Can't you simplify this markup to only have:

```jsx
<div className={classes}>
  {cardToEmoji(type)}
</div>
```

It would also simplify your CSS as `Card__Container` would be a centered flexbox container and you won't need the `Card_AspectRatio` and `Card__Content` anymore.